lib/engine/auto_router.rb

```diff
@@ -64,18 +61,13 @@ def compute(corporation, **opts)
         node_abort = false
 
         node.walk(corporation: corporation, skip_paths: skip_paths) do |_, vp|
-          next :abort if node_abort
-
-          paths = vp.keys
-
-          if Time.now - node_now > node_timeout
-            # TODO: this is not a complete node.walk abort, find somthing more than :abort but less than break.
-            # Or wait til node.walk speed is improved and this may become less important
+          if node_abort || Time.now - node_now > node_timeout
```

Comment:
Once node timeout hits, node_abort is true many times through this node.walk.  So any code that executes when node_abort true is unnecessary.  Granted you're only executing a few lines, but this change risks losing that knowledge "don't do work if node_abort true".

I had a `puts "node_abort in node.walk" if node_abort `and it would print hundreds of times in some cases.